cpp/include/cudf/stream_compaction.hpp

```diff
@@ -149,8 +149,8 @@ enum class duplicate_keep_option {
 std::unique_ptr<experimental::table>
     drop_duplicates(table_view const& input,
                     std::vector<size_type> const& keys,
-                    duplicate_keep_option const& keep,
-                    bool const& nulls_are_equal = true,
+                    duplicate_keep_option const keep,
+                    bool const nulls_are_equal = true,
```

Comment:
I think the const is really unnecessary on a non-reference parameter. Also need an enum class for this. Suggest:

``` 
enum class null_equality : bool  {
  EQUAL,   /// nulls compare equal
  UNEQUAL /// nulls compare unequal
};

```suggestion
                    null_equality_option nulls_equal = null_equality_option::EQUAL,
```

Reply:
updated this enum in cpp and cython